tests/unit/schema/test.version.js

```diff
@@ -1,31 +0,0 @@
-import cloneDeep from 'lodash.clonedeep';
-
-import { validateAddon } from 'schema/validator';
-
-import { validManifest } from './helpers';
-
-describe('/version', () => {
-  it('should be invalid due to invalid version', () => {
-    const manifest = cloneDeep(validManifest);
-    manifest.version = '01';
-    validateAddon(manifest);
-    expect(validateAddon.errors.length).toEqual(1);
-    expect(validateAddon.errors[0].instancePath).toEqual('/version');
-  });
-
-  it('should be invalid due to missing version', () => {
-    const manifest = cloneDeep(validManifest);
-    manifest.version = undefined;
-    validateAddon(manifest);
-    expect(validateAddon.errors.length).toEqual(1);
-    expect(validateAddon.errors[0].instancePath).toEqual('');
-    expect(validateAddon.errors[0].params.missingProperty).toEqual('version');
-  });
```

Comment:
was this test for the missing version property redundant?

Reply:
moved to `tests/unit/test.utils.js`